Bottom: a52fff43a00ef7782f2bed16ecc75238cfab664d
Top:    7cc083b4c8358004bd13184fa90ac2cdb4c48c41
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-02 13:02:22 +0000

hid/common: Fix including draw_helpers.h from draw_helpers.c

The prototype for hid_common_fill_pcb_pv() was incorrect. To avoid this
kind of mistake in the future, include the header file from the .c file.


---

diff --git a/src/hid/common/draw_helpers.c b/src/hid/common/draw_helpers.c
index f6fbe71..02473be 100644
--- a/src/hid/common/draw_helpers.c
+++ b/src/hid/common/draw_helpers.c
@@ -4,6 +4,7 @@
 #include "data.h" /* For global "PCB" variable */
 #include "rotate.h" /* For RotateLineLowLevel() */
 #include "polygon.h"
+#include "draw_helpers.h"
 
 
 static void
